raven/processes/wps_generic_shape_properties.py

```diff
@@ -75,6 +75,8 @@ def _handler(self, request, response):
                     for feature in src:
                         geom = shape(feature['geometry'])
 
+                        multipolygon_check(geom)
```

Comment:
Is this purely for debugging purpose ? It only logs information.

Reply:
At the moment, yes. It's written for cases where the algorithm can only handle a polygon and not a multipolygon. Some of the processes we have are sensitive to this distinction so this idea is to log this information so that the user understands why something has failed.